engine/build.gradle

```diff
@@ -33,6 +36,7 @@ dependencies {
     // For the stable and dev app flavours we're using the library
     // build published at Maven Central.
     stableImplementation 'org.ooni:oonimkall:2022.12.08-092817'
+    dwImplementation 'org.ooni:oonimkall:2022.12.08-092817'
```

Comment:
❗ we need to update this once we merge https://github.com/ooni/probe-android/pull/552